staff/dbenavente/03-and-more/1.js

```diff
@@ -0,0 +1,17 @@
+/*
+randomLargest()
+Randomizes three numbers in range (0.. 100) and prints the largest one.
+*/
+
+function randomLargest(){
+  var randomOne = Math.floor(Math.random()*(101))
```

Comment:
Have you tried  `Math.ceil(Math.random()*100)`? (you also don't need parenthesis)